packages/site/cms-site/src/graphQLFetch/graphQLFetch.ts

```diff
@@ -30,9 +30,32 @@ function graphQLHeaders(previewData?: SitePreviewData) {
     return headers;
 }
 
-//from graphql-request https://github.com/jasonkuhrt/graphql-request/blob/main/src/raw/functions/gql.ts
+// Adapted from https://github.com/jasonkuhrt/graffle/blob/main/src/layers/6_helpers/gql.ts
```

Comment:
@nsams graphql-request parse the document with `graphql`, should we do this as well?

Reply:
what would be the benefit of doing this?

Reply:
I'm not sure, possibly the validity of the GraphQL syntax would be checked before even sending it to the API.

Reply:
Hm, the server checks it anyway, only for that i would not do it